core/src/types.ts

```diff
@@ -88,6 +88,11 @@ export declare interface RegularCalleeConfig {
     route: string[];
 }
 
+export declare interface AutoRouterCalleeConfig {
+    callee: RegularCalleeConfig['callee'];
+    automaticRouter: true;
+}
```

Comment:
If you modify this to be 
```suggestion
export declare interface AutoRouterCalleeConfig {
    callee: 'UniswapV3Callee';
    automaticRouter: true;
}
```